AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/ErrorMessages.dfy

```diff
@@ -56,4 +56,7 @@ module {:options "/functionSyntax:4" } KeyStoreErrorMessages {
 
   const INVALID_HIERARCHY_VERSION :=
     "Invalid hierarchy version. Expected version 1 or 2."
+
+  const MD_DIGEST_SHA_NOT_MATCHED :=
+    "Decrypted md digest SHA does not match md digest sha from the table."
```

Comment:
Blocking: I see what you are doing here, but no one outside of our team will have the context to understand this error message.
```suggestion
    "This Branch Key item has failed the authentication check. Either it has been tampered with or the wrong 'Logical Key Store Name' has been provided."
```